test/datasets/speechcommands_test.py

```diff
@@ -0,0 +1,74 @@
+import os
+import random
+
+from torchaudio.datasets import speechcommands
+
+from ..common_utils import (
+    TempDirMixin,
+    TorchaudioTestCase,
+    get_whitenoise,
+    normalize_wav,
+    save_wav,
+)
+
+
+class TestSpeechCommands(TempDirMixin, TorchaudioTestCase):
+    backend = "default"
+
+    root_dir = None
+    samples = []
+
+    @classmethod
+    def setUp(cls):
+        cls.root_dir = cls.get_base_temp_dir()
+        dataset_dir = os.path.join(
+            cls.root_dir, speechcommands.FOLDER_IN_ARCHIVE, speechcommands.URL
+        )
+        os.makedirs(dataset_dir, exist_ok=True)
+        sample_rate = 16000  # 16kHz sample rate
+        random.seed(0)
```

Comment:
`random` is no longer used, and leaving it here will be confusing later. Can you remove it?